src/sqlfluff/dialects/dialect_ansi.py

```diff
@@ -999,6 +1001,8 @@ class SelectClauseSegment(BaseSegment):
         terminator=OneOf(
             "FROM",
             "LIMIT",
+            "WHERE",
```

Comment:
Is it significant that above, we see `ORDER`, then `WHERE`, but here, they are reversed?

Reply:
Not significant, but I'll make them the same to avoid confusion.